webgame-game/src/gridworld.rs

```diff
@@ -191,11 +211,17 @@ fn visualize_agent<T: Component>(
 
 const AGENT_SPEED: f32 = GRID_CELL_SIZE * 2.;
 
-/// Allows the player to move the Player agent around.
-fn move_player(
+/// Holds the next action for an agent.
```

Comment:
Instead of directly setting the positions of agents via the keyboard, all agents now have a `NextAction` component that determines what they'll do next. This allows us to control agents programmatically from Python.